### DISCUSSION TITLE
Increase the `author` field size for tree tables

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5914] [Feature Requests]
There are numerous taxa records in Catalogue of Life that exceed the `author` field size.

For example:
https://www.catalogueoflife.org/data/taxon/33PP6

_Cystobasidium calyptogenae_ (Nagah., Hamam., Nakase & Horikoshi) Yurkov, Kachalkin, H.M. Daniel, M. Groenew., Libkind, V. de García, Zalar, Gouliam., Boekhout & Begerow

The `author` value for this record is as follows:
```
(Nagah., Hamam., Nakase & Horikoshi) Yurkov, Kachalkin, H.M. Daniel, M. Groenew., Libkind, V. de García, Zalar, Gouliam., Boekhout & Begerow
```

it is 140 characters. The `author` field only supports 128 characters. ([taxon for reference](https://files.specifysoftware.org/schema/version/2.10/#taxon))

**Comment by g1rly-c0d3r** [comment]
linked in https://github.com/specify/specify7/issues/8193
